### DISCUSSION TITLE
Everything seems to be installed. But it didn't work

### DISCUSSION THREAD

**Discussion by kpiboboi** [samveen/mt7902-dkms#10] [General]
The installation is successful, but the Wi-Fi card does not see the card

Laptop model: Asus Vivobook Go E1504FA
OS: 6.8.0-31-generic #31-Ubuntu SMP PREEMPT_DYNAMIC x86_64 x86_64 x86_64 GNU/Linux
CPU: AMD Ryzen 5 7520U with Radeon Graphics
WLAN name: Network controller: MEDIATEK Corp. Device 7902

**Comment by samveen** [comment]
@kpiboboi this experiment is stalled since last week as I haven't had the time to rebase the code on the mt7925 as compared to the mt7921, Please see #8 for more details.
